src/style.css

```
@@ -0,0 +1,84 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: 300px;
+  min-width: 300px;
+  line-height: 300px;
+  color: white;
+
+  text-align: center;
+  flex-grow: 1;
+}
+
+.block1 {
+  background-color: rgb(255, 0, 0);
+}
+
+.block2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.block3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.block4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.block5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.block6 {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (min-width: 600px) {
+  .block3 {
+    order: 1;
+  }
+
+  .block5 {
+    order: 2;
+
+  }
+
```

```suggestion
  .block5 {
    order: 2;
  }

```

Не залишай пустий рядок в CSS правилах

---

прийняв :))))